pkgs/by-name/lu/lunarvim/package.nix

```diff
@@ -0,0 +1,131 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, fetchpatch
+, makeWrapper
+, curl
+, fd
+, fzf
+, git
+, gnumake
+, gnused
+, gnutar
+, gzip
+, lua-language-server
+, neovim
+, nodePackages
+, ripgrep
+, tree-sitter
+, unzip
+, nvimAlias ? false
+, viAlias ? false
+, vimAlias ? false
+, globalConfig ? ""
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "lunarvim";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "LunarVim";
+    repo = "LunarVim";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-z1Cw3wGpFDmlrAIy7rrjlMtzcW7a6HWSjI+asEDcGPA=";
+  };
+
+  # Pull in the fix for Nerd Fonts until the next release
+  patches = [
+    (
+      fetchpatch {
+        url = "https://github.com/LunarVim/LunarVim/commit/d187cbd03fbc8bd1b59250869e0e325518bf8798.patch";
+        sha256 = "sha256-ktkQ2GiIOhbVOMjy1u5Bf8dJP4SXHdG4j9OEFa9Fm7w=";
+      }
+    )
+  ];
+
+  nativeBuildInputs = [
+    gnused
+    makeWrapper
+  ];
+
+  buildInputs = [
+    curl
+    fd
+    fzf
+    git
+    gnumake
+    gnutar
+    gzip
+    lua-language-server
+    neovim
+    nodePackages.neovim
+    ripgrep
+    tree-sitter
+    unzip
+  ];
+
+  dontBuild = true;
+
+  installPhase = ''
```

Comment:
Wouldn't it make debugging the derivation easier to run these steps in a build phase ? install could be just moving the folder.

Reply:
Let me know if this is what you meant.